frame/example/src/lib.rs

```diff
@@ -431,7 +431,7 @@ decl_module! {
 		// - MUST NOT PANIC: Under no circumstances (save, perhaps, storage getting into an
 		// irreparably damaged state) must this function panic.
 		// - NO SIDE-EFFECTS ON ERROR: This function must either complete totally (and return
-		// `Ok(())` or it must have no side-effects on storage and return `Err('Some reason')`.
+		// `Ok(None.into())` or it must have no side-effects on storage and return `Err('Some reason')`.
```

Comment:
eventually if this PR is approved, these docs could use an update.

Reply:
Absolutely. I added a task item for that to the PR.